tumblr-photo-video-ripper.py

```diff
@@ -20,11 +19,16 @@
 START = 0
 
 # Numbers of photos/videos per page
-MEDIA_NUM = 50
+MEDIA_NUM = 20
 
 # Numbers of downloading threads concurrently
 THREADS = 10
 
+# just a test apikey
+API_KEY = "lmvVU5ExdfFZPyGOv0gCknJ2r1UnQEIZTYAYoDhKrq7eJdCn2o"
+
+# enum(posts,likes)
+POST_TYPE="likes"
```

Comment:
Better add some doc here.

For most common usage, `POST_TYPE = ("likes", "posts")`